### PR TITLE
Seedlist for state channels

### PR DESCRIPTION
**Pull request by TheMMaciek**

### CODE REVIEW

File: kubernetes/data/state-channel-seedlist.csv [hunk 1]
```diff
@@ -0,0 +1,10 @@
+DAG7QrKeoHbYezJ6YHUCt7PJDixWgxM4vfKz8qVT
+DAG3FJFFsjzYeSNVeL3cykLqWbijkq25K6Eg23yp
+DAG5kq5J6gudAbyGsgqPReGz8ZDRM94UoPwCFSem
+DAG3ysjCYZCWoTw3SKoK13McfdA5LZWYBr7ue47d
+DAG7cCCNgP8D5rdEznhSNXpAfQ2KhuScxhg1YhVX
+DAG2T7jB5kXpQMEpDErB2NcoQfeJP7DwbueK3A5i
+DAG16Q8MxAvihBbFk8wuSaCVzieq75zZA3mh414r
+DAG6wD9h8NE6qDuVT8ZtJFaU3ZzD9BqBjWD8GDee
+DAG0R1CMwfLPNp4qkA2TTqkcKWp3mC7U4ZMD5n1B
+DAG7gcb396YoLVzNhCEF2nrB79uwV3wr4BvXSU9f
```
Comment (marcinwadon) [inline kubernetes/data/state-channel-seedlist.csv:10]: are you sure it should be a DAGAddress, not a node id (so the public key)?

File: modules/shared/src/test/resources/logback-test.xml
Comment (marcinwadon) [inline modules/shared/src/test/resources/logback-test.xml:1]: is it a leftover or is it required?

File: modules/core/src/main/scala/org/tessellation/cli/method.scala [hunk 2]
```diff
@@ -122,6 +124,7 @@ object method {
     environment: AppEnvironment,
     snapshotConfig: SnapshotConfig,
     seedlistPath: Option[Path],
+    stateChannelSeedlistPath: Path,
```
Comment (kpudlik) [inline modules/core/src/main/scala/org/tessellation/cli/method.scala:127]: Isn't it `seedListPath === stateChannelSeedlistPath` for Global L0?

**Review by kpudlik — CHANGES_REQUESTED** [review]

File: kubernetes/data/state-channel-seedlist.csv (hunk 1 above)
Comment (TheMMaciek) [reply kubernetes/data/state-channel-seedlist.csv:10]: Yes it needs to be a DAGAddress. Currently it's derived from public key, but we will add functionality to derive the address from state channel genesis.

File: modules/core/src/main/scala/org/tessellation/cli/method.scala (hunk 2 above)
Comment (TheMMaciek) [reply modules/core/src/main/scala/org/tessellation/cli/method.scala:127]: No, state channel seedlist path is a list of DAGAddresses.

File: modules/shared/src/test/resources/logback-test.xml
Comment (TheMMaciek) [reply modules/shared/src/test/resources/logback-test.xml:1]: It disables logs in tests hence makes test runtime more readable.

File: modules/sdk/src/main/scala/org/tessellation/sdk/cli/StateChannelSeedlist.scala [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+import org.tessellation.ext.decline.decline._
+
+import com.monovore.decline.Opts
+import fs2.io.file.Path
+
+object StateChannelSeedlist {
+
+  val opts: Opts[Path] = Opts
+    .option[Path]("state-channel-seedlist", "List of allowed state channel addresses")
```
Comment (tbekas) [inline modules/sdk/src/main/scala/org/tessellation/sdk/cli/StateChannelSeedlist.scala:11]: Consider adding the env variable as well

File: modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/genesis/Loader.scala [hunk 4]
```diff
@@ -21,7 +21,7 @@ object Loader {
           decodeWithoutHeaders[GenesisCSVAccount]()
         )
         .map(_.toGenesisAccount)
-        .map(_.bimap(e => new RuntimeException(e), identity))
+        .map(_.leftMap(new RuntimeException(_)))
```
Comment (tbekas) [inline modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/genesis/Loader.scala:24]: 👍

File: modules/core/src/main/scala/org/tessellation/cli/method.scala (hunk 2 above)
Comment (kpudlik) [reply modules/core/src/main/scala/org/tessellation/cli/method.scala:127]: But isn't it still `seedList.map(_.toAddress) === stateChannelSeedlist` for Global L0?

**Review by tbekas — DISMISSED** [review]

**Review by marcinwadon — APPROVED** [review]

**Review by kpudlik — DISMISSED** [review]

File: modules/shared/src/test/resources/logback-test.xml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<configuration>
```
Comment (marcinwadon) [inline modules/shared/src/test/resources/logback-test.xml:1]: why is it needed?

**Review by marcinwadon — DISMISSED** [review]
LGTM

**Review by kriscieslak — APPROVED** [review]

**Pull request merged**